app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,23 @@
+<header class="header">
+  <nav>
+    <div class="header__backbtn">
+    <% if signout == "yes" %>
+    <div class="logout">
+      <%= link_to '', destroy_user_session_path, method: :delete, class: "fas fa-sign-out-alt fa-lg text-reset text-decoration-none" %>
+      <p class="lg-o">Log out</p>
+    </div>
+    <% else %>
+      <%= link_to '', path, class: "fas fa-arrow-left fa-lg text-reset text-decoration-none"%>
+    <% end %>
+    </div>
+    <p class="text-reset fs-3"><%= var%></p>
```

Comment:
- [ ] The button to add `new transaction` or `new group` should be at the bottom of the page as per design. Please fix it. Currently, you have added in the header.

### Your design (Button to add transactions is in the header but it should be at the bottom)
![Screenshot from 2022-02-03 03-08-50](https://user-images.githubusercontent.com/22434660/152244991-592a6aac-7093-4c48-aa33-fe8ca307ff06.png)

### Your design (Button to add Group is in the header but it should be at the bottom)
![Screenshot from 2022-02-03 03-09-57](https://user-images.githubusercontent.com/22434660/152245265-2ed16b72-e882-4bcb-bd67-b8e51a665bf6.png)


### Given design
![Screenshot from 2022-02-03 03-28-53](https://user-images.githubusercontent.com/22434660/152244976-0b967431-1175-46c6-bab1-c325f7569a4f.png)

Reply:
According to the design on Behance the button is placed on top.